backend/webapi/Features/EndorsementRequests/Create.cs

```diff
@@ -93,24 +114,26 @@ public async Task<Model> HandleAsync(Command command)
             this.context.EndorsementRequests.Add(request);
             await this.context.SaveChangesAsync();
 
-            await this.SendEndorsementRequestEmailAsync(request.RecipientEmail, request.Token, partyName);
+            await this.SendEndorsementRequestEmailAsync(request.RecipientEmail, request.Token, partyName, isUnlicensed);
 
             return new(false);
         }
 
-        private async Task SendEndorsementRequestEmailAsync(string recipientEmail, Guid token, string partyName)
+        private async Task SendEndorsementRequestEmailAsync(string recipientEmail, Guid token, string partyName, bool isUnlicensed)
         {
             string url = this.applicationUrl.SetQueryParam("endorsement-token", token);
             var link = $"<a href=\"{url}\" target=\"_blank\" rel=\"noopener noreferrer\">this link</a>";
             var pidpSupportEmail = $"<a href=\"mailto:{EmailService.PidpEmail}\">{EmailService.PidpEmail}</a>";
             var pidpSupportPhone = $"<a href=\"tel:{EmailService.PidpSupportPhone}\">{EmailService.PidpSupportPhone}</a>";
+            var templatetext = isUnlicensed ? "You are receiving this email because a user is requesting to endorse you."
```

Comment:
```suggestion
            var templatetext = isUnlicensed ? "You are receiving this email because a user has started an endorsement request with you."
```